docs/source/user_section/installation/install_on_windows.rst

```diff
@@ -0,0 +1,140 @@
+***********************
```

Comment:
I would mention possible problems with conda installation if the path has spaces in it.

Reply:
Done here: 18c7d78d82f45626ff350eda5b758557ffb63187